treehasher.go

```diff
@@ -10,6 +10,7 @@ var nodePrefix = []byte{1}
 
 type treeHasher struct {
 	hasher hash.Hash
+	ph     []byte
```

Comment:
Can we give this a slightly more descriptive name or add a comment on top?

Reply:
Yeah, I changed it to `zeroValue`.